pkg/config/manager.go

```diff
@@ -26,6 +26,16 @@ type PostgresConnection struct {
 	SslMode      string `yaml:"ssl_mode" json:"ssl_mode" default:"disable"`
 }
 
+type MsSQLConnection struct {
+	Name     string `yaml:"name" json:"name"`
+	Username string `yaml:"username" json:"username"`
+	Password string `yaml:"password" json:"password"`
+	Host     string `yaml:"host"     json:"host"`
+	Port     int    `yaml:"port"     json:"port"`
+	Instance string `yaml:"instance" json:"instance"`
```

Comment:
what's an instance? is there a way we can eliminate this and make it similar to other db connections? is that schema maybe? thinking outloud.

Reply:
Instance is an [MSSQL concept](task.Name, query) and equates to `database` in other RDBMS. Its unfortunate because there is also `database` and even `schema` it seems, although I don't see a way to specify the latter. Do you think we can handle them differently?